doc/content/concepts/email-templates.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Email Templates"
+description: ""
+weight: 15
+---
+
+## What is it?
+
+Email templates define the contents of the emails that The Things Stack sends to its users. They allow users to override the default contents of the emails with their own custom contents.
+
+## Who is it for?
+
+Email templates are targeted at network administrators that would like to customize the emails that the stack sends to its users.
+
+### Typical use cases
+
+1. Adding additional styling to the default text-only emails that the stack sends.
+2. Translating of the emails the stack sends to its users.
+ 
+## How does it work?
+
+Email templates override the default emails that the stack sends by providing custom template files written in Go's [html/template](https://golang.org/pkg/html/template/) format using [fetchers]({{< ref "/concepts/fetchers.md" >}}). The templates are retrieved by the stack when the stack sends the first email of a certain type, and then are cached until the stack is restarted. See [Available Templates]({{< ref "/reference/email-templates/available.md" >}}) and [Overriding Default Templates]({{< ref "/reference/email-templates/overriding.md" >}}).
```

Comment:
"using fetchers" sounds a bit strange

Reply:
I think the internal workings belong in Reference; Concepts are short and concise, focusing on how it works for the end user